pyproject.toml

```diff
@@ -211,18 +212,99 @@ supported_nautobot_versions = [
     "2.0.0"
 ]
 
-[tool.pydocstyle]
+[tool.ruff]
+line-length = 120
+target-version = "py38"
+
+[tool.ruff.lint]
+select = [
+    "D",  # pydocstyle
+]
+ignore = [
+    # warning: `one-blank-line-before-class` (D203) and `no-blank-line-before-class` (D211) are incompatible.
+    "D203",  # 1 blank line required before class docstring
+
+    # D212 is enabled by default in google convention, and complains if we have a docstring like:
+    # """
+    # My docstring is on the line after the opening quotes instead of on the same line as them.
+    # """
+    # We've discussed and concluded that we consider this to be a valid style choice.
+    "D212",  # Multi-line docstring summary should start at the first line
+    "D213",  # Multi-line docstring summary should start at the second line
+
+    # Produces a lot of issues in the current codebase.
+    "D401",  # First line of docstring should be in imperative mood
+    "D407",  # Missing dashed underline after section
+    "D416",  # Section name ends in colon
+
+    # Package specific
+    "D417",  # Missing argument descriptions in Docstrings
+]
+
+[tool.ruff.lint.pydocstyle]
 convention = "google"
-inherit = false
-match = "(?!__init__).*\\.py"
-match-dir = "(?!tests|migrations|development)[^\\.].*"
-# D212 is enabled by default in google convention, and complains if we have a docstring like:
-# """
-# My docstring is on the line after the opening quotes instead of on the same line as them.
-# """
-# We've discussed and concluded that we consider this to be a valid style choice.
-add_ignore = "D212"
+
+[tool.ruff.per-file-ignores]
+"nautobot_chatops/migrations/*" = [
+    "D",  # pydocstyle
+]
+"nautobot_chatops/tests/*" = [
+    "D",  # pydocstyle
+]
 
 [build-system]
 requires = ["poetry_core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.towncrier]
+package = "nautobot"
+directory = "changes"
+filename = "docs/admin/release_notes/version_X.Y.md"
```

Comment:
Is this supposed to be changed before merging?

Reply:
Nope, AFAIK renaming the file after it's generated should be enough.